### DISCUSSION TITLE
Expand icon options, allow for emoji and standard characters as button

### DISCUSSION THREAD

**Discussion by MrComputron** [chrisgurney/obsidian-note-toolbar#42] [Ideas]
I would find it very useful if the icon list was expanded or at least allow for selecting any keyboard character including emoji as the button.

**Comment by chrisgurney** [comment]
You can put emojis in the title of an item, and it will work. Just leave the icon blank (not set).

**Reply by MrComputron** [reply]
Thanks for the tip!

**Reply by MrComputron** [reply]
Works like a charm!
